### PR TITLE
added showerthoughts entry

### PR DESCRIPTION
**Pull request by PTraum**

### CODE REVIEW

**Comment by PTraum** [comment]
Please review this, @casey-collab

File: english/showerthoughts/showerthoughts.md [hunk 1]
```diff
@@ -0,0 +1 @@
+[showerthoughts](https://www.reddit.com/r/Showerthoughts/)
```
Comment (casey-collab) [inline english/showerthoughts/showerthoughts.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by PTraum** [comment]
combined both new entires in marshmallow.md

**Comment by PTraum** [comment]
Please review this, @casey-collab

File: english/showerthoughts/showerthoughts.md (hunk 1 above)
Comment (casey-collab) [inline english/showerthoughts/showerthoughts.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by PTraum** [comment]
removed link.
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :ferris_wheel:

**Pull request merged**